tlg-base-config/azuredeploy.json

```diff
@@ -0,0 +1,568 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "localAdmin": {
+            "type": "string",
+            "defaultValue": "localAdmin",
+            "metadata": { 
+	            "description": "The 'localAdmin' account on each of the Virtual Machines (also becomes EA/DA on DC Promotion of DC1)" 
+            }
+        },
+        "localAdminPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'localAdmin' account on each of the Virtual Machines" 
+            }
+        },
+        "safemodeAdmin": {
+            "type": "string",
+            "defaultValue": "safemodeAdmin",
+            "metadata": { 
+	            "description": "The safemodeAdmin account for the Domain" 
+            }
+        },
+        "safemodeAdminPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'safemodeAdmin'" 
+            }
+        },
+        "genericUser": {
+            "type": "string",
+            "defaultValue": "User1",
+            "metadata": { 
+	            "description": "The account to be created and promoted to EA/DA" 
+            }
+        },
+        "genericUserPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'genericUser' account" 
+            }
+        },
+        "newStorageAccountName": { 
+	        "type": "string", 
+	        "metadata": { 
+	            "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed. (Length: 3-24; Case: Lower; Valid: Alphanumeric; Unique: Global)" 
+            } 
+        }
+    },
+    "variables": {
+        "githubroot": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/tlg-base-config/dsc/",
```

Comment:
See:
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc

For artifact use